test.py

```diff
@@ -297,9 +308,18 @@ def run_tests(
     options=[],
     interop_tests=False,
     workers=None,
+    only_pattern=None,
 ):
     if interop_tests:
-        legate_tests.extend(glob.glob("tests/interop/*.py"))
+        legate_tests.extend(glob.glob("tests/interop/*.py"))  # noqa: F823
+
+    if only_pattern is not None:
```

Comment:
Let's not repeat this, unless there's a reason good reason to do so. Can't you factor this and the same code above out?

Reply:
Sure, done